tools/generate_from_protos.sh

```diff
@@ -4,7 +4,7 @@ set -e
 
 script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-proto_dir="${script_dir}/../proto/protos"
+proto_dir="${script_dir}/../src/backend/proto/protos"
```

Comment:
Is this correct?

Reply:
Yes, this is fix. Apparently, protos was moved before. 
gRPC headers and SRC generated correctly now

Reply:
oh ok!